modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -102,7 +102,7 @@ The following examples show situations that require configuring a network interf
 
 [IMPORTANT]
 ====
-You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
+Only a bare-metal platform supports using a `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
```

Comment:
Not all baremetal supports using 'br-ex' when configuring DNS resolvers, only a baremetal with nmstate-configuration-based br-ex supports that, if the br-ex is configure-ovs-based, it will not support that, because there will be conflict between configure-ovs and nmstate.

Reply:
Thanks, @qiowang721 .

How about:

> A bare-metal platform with an NMState configuration that sets a `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface only supports configuring DNS resolvers. If you also used `configure-ovs.sh` to set a `br-ex` bridge for your network, this can cause conflicting with the `br-ex` bridge that you set by using the NMState configuration method.